### PR TITLE
Whatsnew for May 4th

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) updates the whatsnew yaml file with latest changes.

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,56 @@ description: |
...
+  type: 'Major Update'
+  date: April 29, 2020
+  link: https://github.com/magento/devdocs/pull/7118
+- description: Updated the [PHP requirements](https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html#php)
+    and [packages](https://devdocs.magento.com/guides/v2.3/release-notes/packages-open-source.html).
+  versions: 2.3.5
+  type: 'Major Update'
+  date: April 29, 2020
+  link: https://github.com/magento/devdocs/pull/7119
+- description: 'MISSING whatsnew in the PR #7131: "Magento 2.3.5 Release Note hotfix
```
Comment (dshevtsov) [inline src/_data/whats-new.yml:40]: Invalid description

File: src/_data/whats-new.yml [hunk 2]
```diff
@@ -37,8 +37,7 @@ entries:
   type: 'Major Update'
   date: April 29, 2020
   link: https://github.com/magento/devdocs/pull/7119
-- description: 'MISSING whatsnew in the PR #7131: "Magento 2.3.5 Release Note hotfix
-    update"'
+- description: 'Added information about the April 30th Resend account confirmation email link issue patch'
```
Comment (dshevtsov) [inline src/_data/whats-new.yml:40]: This text must be added to https://github.com/magento/devdocs/pull/713. Do not add changes manually to descriptions in this file.

**Comment by dshevtsov** [comment]
I feel like I should explain why I run `since='apr 23'`, whereas the last update date was April 27.
Last time the file was updated specifically for 2.3.5 release. The entries were generated from `2.3.5-develop` branch only. The latest entry date prior the 2.3.5 updates were April 23. That's where I took the date.

**Comment by dshevtsov** [comment]
Also, I noticed that no _Technical_ updates were added. When I generate the file, I get 5 _Technical_ entries with descriptions.
If you do not get them in your system, check `labels` in your configuration (`.whatsup.yml`), the spelling must be identical to same labels on GitHub.

**Comment by dobooth** [comment]
running tests

**Pull request merged**